### ISSUE TITLE
Object Marker: On changing the theme to High contrast white/Black theme, object markers used in another component are not adapted to it.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3838]
Description: On changing the theme to High contrast white/Black theme, colored icons within the list are not adapted to it.

Expected: All UI elements shall adapt to high contrast white/black theme, when applied

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98775181-9acbf080-2412-11eb-830c-77305ad71dfb.png)

In high contrast themes:

![image](https://user-images.githubusercontent.com/32538291/98775184-9dc6e100-2412-11eb-935e-095e4585d788.png)

![image](https://user-images.githubusercontent.com/32538291/98775214-b20ade00-2412-11eb-91c8-d3464af102e7.png)

**Comment by sKudum** [comment]
Icons used here are Object marker. Issue needs be raised for object markers. Hence reframing the sentence and  assigning it to Lok.

**Comment by Lokanathan-k** [comment]
As per fiori Specs (https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2060036541#ObjectMarker(Fiori3)-Specifications(CozyandCompact))the labels are always blue in color and its working as expected. and there is no differentiation of colours to adapt to the high contrast.

**Comment by sKudum** [comment]
Thank You lok. We can close it then Puru.

**Comment by puru-hk** [comment]
Tested as per the fiori3 link https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2095455756 it is working as expected.
